src/tools/swix/swcore/CompilerBackend/Nuget/NugetBackendCompiler.cs

```diff
@@ -53,7 +53,10 @@ public override void Generate(IEnumerable<Intermediate> intermediates, string ou
                 return;
             }
 
-            FileTransfer packageTransfer = FileTransfer.Create(null, Path.GetTempFileName(), outputPath, "NugetPackage", true);
+            int lastIndex = outputPath.LastIndexOf('.');
```

Comment:
Any ideas on how to do that?  The version is specified in the manifest, I don't know about the .targets having to know how to read the manifest.